fasta/fasta.go

```diff
@@ -103,6 +103,22 @@ func Write(filename string, records []Fasta) {
 	exception.PanicOnErr(file.Close())
 }
 
+// Write a fasta to input filename. Output fastas have line length of 50.
+func WriteMap(filename string, records FastaMap) {
```

Comment:
non deterministic output order from this function? Should FastaMap map chrom names to structs, including the sequence and the index?

Reply:
I think we should just stick with Write unless there is a specific reason to write from a map. I think it would be good practice to always keep around the slice of ordered fasta if you ever plan to write them to a file. I don't love the idea of mapping to structs.

Reply:
To address these comments and others, I think I will write a function that copies seq back from a map back to a slice, maintaining the order of the slice, but updating with seq from the map.  Or maybe I will just make a note in the code that this would be needed for future extensions to handle insertions.